website/addons/figshare/views/config.py

```diff
@@ -38,16 +38,23 @@ def figshare_config_put(node_addon, auth, **kwargs):
     fields = request.json.get('selected', {})
     node = node_addon.owner
 
-    folder = {
-        'name': fields.get('name') or '',
-        'id': fields.get('id') or None,
-        'type': fields.get('type') or None,
-    }
-    node_addon.update_fields(folder, node, auth)
+    name = fields.get('name') or ''
+    fs_id = fields.get('id') or None
```

Comment:
Also, to get nitpicky... let's avoid acronyms in variable names. `figshare_id`, `figshare_type` would be an improvement.